core/src/main/java/com/yahoo/ycsb/Client.java

```diff
@@ -469,6 +475,10 @@ public int getOpsTodo()
    */
   public static final String MAX_EXECUTION_TIME = "maxexecutiontime";
 
+  /**
+   * All keys for configuring the tracing system start with this prefix.
+   */
+  private static final String HTRACE_KEY_PREFIX="htrace.";
```

Comment:
consider renaming this to "htrace.ycsb_core", I can see in the future some DB binding may be interested in using htrace too.